bundle/run.sh

```diff
@@ -476,11 +476,11 @@ done
 ## configuration
 cat /configs/identity-store.yaml | yq e "\
   .apis.grpc.address = \"${IDENTITY_STORE_ADDRESS}\" |
-  .apis.grpc.authorization.http.tls.useSystemCAPool = true |
+  .apis.grpc.authorization.http.tls.useSystemCaPool = true |
   .apis.grpc.authorization.ownerClaim = \"${OWNER_CLAIM}\" |
   .apis.grpc.authorization.audience = \"${SERVICE_OAUTH_AUDIENCE}\" |
   .apis.grpc.authorization.authority = \"https://${OAUTH_ENDPOINT}\" |
-  .clients.storage.mongoDB.uri = \"${MONGODB_URI}\" |
+  .clients.storage.mongoDb.uri = \"${MONGODB_URI}\" |
```

Comment:
Heh, this one is pretty specific. Product is named `mongoDB`, I wouldn't change this one to `mongoDb`, it just looks wierd as my eyes are used to mongoDB as it's written like this everywhere :)

Reply:
We will keep `mongoDB`.